include/sound/sof.h

```diff
@@ -122,6 +122,10 @@ struct sof_dev_desc {
 	/* defaults for no codec mode */
 	const char *nocodec_tplg_filename;
 
+	/* information on supported IPCs */
```

Comment:
you mean SOF_INTEL_IPC4 instead of cavs IPC in the commit message?

Reply:
yes, nice catch. fixed now.